### PR TITLE
add `_TZE200_rjxqso4a` as `ZC-HM`

### PR DESCRIPTION
**Pull request by nklebedev**
add Moes ZC-HM carbon monoxide sensor
closes https://github.com/Koenkk/zigbee2mqtt/issues/19011
adding image in https://github.com/Koenkk/zigbee2mqtt.io/pull/2277

### CODE REVIEW

**Comment by nklebedev** [comment]
[database.db.entry.json](https://github.com/Koenkk/zigbee-herdsman-converters/files/13034652/database.db.entry.json)

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -737,6 +737,28 @@ const definitions: Definition[] = [
         toZigbee: [],
         exposes: [e.carbon_monoxide(), e.co()],
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE200_rjxqso4a']),
```
Comment (Koenkk) [inline src/devices/tuya.ts:741]: Please move it to `moes.js` and remove the `whitelabel` (update `model`/`vendor` instead)

Comment (nklebedev) [reply src/devices/tuya.ts]: ok

**Comment by Koenkk** [comment]
Thanks!

**Comment by mtausk** [comment]
@Koenkk and @nklebedev , guys, there is no mention on how to pair this device with zigbee2mqtt. Do you have some advice?

**Comment by nklebedev** [comment]
> @Koenkk and @nklebedev , guys, there is no mention on how to pair this device (MOES ZC-HM) with zigbee2mqtt. Do you have some advice?

Hi, you need to press the button located on top of the device three times.

**Comment by patrick-foubert-56770** [comment]
Hello added into 1.38.0-1

My states report;

{
    "battery": 100,
    "last_seen": "2024-11-27T17:19:59.059Z",
    "linkquality": 47,
    "self_test_result": "success"
}

No CO PPM...

**Comment by MqlBql** [comment]
Same issue here. Only got battery and link state, no CO or ppm.

**Pull request merged**